tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.targets

```diff
@@ -75,13 +75,11 @@
     </OutputHashTask>
 
     <!-- Run build telemetry -->
-    <BuildTelemetryTask
+    <Exec
```

Comment:
We can use the standard [Exec](https://learn.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2022) task to call the reporter, no need for a custom task.